server/model/index.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+// TODO: needs update when we integrate csv feature
+const data_report_params = schema.object({
```

Comment:
how come these consts are snake case and the others in the file are camel case?

Reply:
thx for pointing that out. Changed the `consts` to use camel case